runtime/bcutil/ClassFileOracle.hpp

```diff
@@ -113,6 +113,16 @@ class ClassFileOracle
 		bool isByteCodeFixupDone;
 	};
 
+	struct RecordComponentInfo
+	{
+		U_16 nameIndex;
+		U_16 descriptorIndex;
+		bool hasGenericSignature;
```

Comment:
move the `hasGenericSignature` member first so the other fields can potentially pack better